sass/_clients.scss

```diff
@@ -1,77 +1,161 @@
-.featured-clients {
+.hero-header {
+    text-align: center;
+    margin-top: 90px;
+    margin-bottom: 81px;
+    color: #000000;
+    width: 100%;
+    max-width: 651px;
+
+
+    @media (max-width: 767px) {
+        max-width: 312px;
+        margin-top: 56px;
+        margin-bottom: 34px;
+    }
+
+    h1 {
+        width: 100%;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 40px;
+        line-height: 120%;
+    }
+
+    p {
+        width: 100%;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 18px;
+        line-height: 120%;
+    }
+}
+
+section {
     display: flex;
-    gap: .5rem;
-    overflow-x: auto;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+.featured-clients {
+    flex-grow: 3;
+    width: 100%;
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-columns: 1fr 1fr 1fr;
+
+
+    @media (max-width: 767px) {
+        width: unset;
+        grid-template-columns: 1fr;
+        grid-auto-flow: row;
+    }
+
+    gap: 50px;
+    max-width: 943px;
 }
 
 .featured-client-card {
     display: flex;
     flex-direction: column;
-    flex: 0 0 250px;
-    border-color: 1px solid rgba(43, 140, 201, .4);
-    background-color: rgba(43, 140, 201, .1);
-    border-radius: 1em;
-    padding: 1em;
+    justify-content: center;
+    align-items: center;
+    padding: 24px;
+    gap: 16px;
+
+    width: 281px;
+    height: 343px;
+
+    border: 1px solid #D2D2D2;
+    border-radius: 16px;
 
     img {
-        height: 160px;
-        display: block;
-        margin: 0 auto;
+        margin: 0;
+        width: 65px;
+        height: 65px;
+        box-sizing: content-box;
     }
 
-    h3 {
-        text-align: center;
+    p {
+        margin: 0;
     }
 
-    .platforms-links {
-        display: flex;
-        flex-wrap: wrap;
-        justify-content: center;
-        margin-top: auto;
+    h3 {
+        margin: 0;
+
+        font-style: normal;
+        font-weight: 700;
+        font-size: 32px;
+        line-height: 120%;
     }
 }
 
+.pills {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    gap: 16px 4px;
+    flex-wrap: wrap;
+}
+
 .platform-pill {
-    margin: .2em;
-    border-radius: 2em;
-    padding: 0 .6em;
-    color: #fff;
-    background-color: rgba(43, 140, 201, .8);
+    background: #E9E9E9;
+    border-radius: 16px;
+    padding: 6px 8px;
```

Comment:
Probably worth using `rem`s here again to make the pills scale with the zoom level